frontend/checkout.phtml

```diff
@@ -81,14 +81,18 @@
 </form>
 
 <script>
-jQuery(document).ready(function( $ ){
-    function showModal(response) {
+jQuery(document).ready(function( $ )
```

Comment:
1. no space around parameters in signatures ^
2. put function opening braces on the same line